### ISSUE TITLE
Feature Request: Replay Button

### ISSUE THREAD

**Issue by sifujerry** [HermesApp/Hermes#318]
The Replay button is kind of new in the Pandora interface. I'd love to see it added to Hermes. 

Love the app!

**Comment by nriley** [comment]
I agree, this would be a great feature to add. I don’t think it would be terribly hard either.

**Comment by laynetrain** [comment]
I came here to request that feature as well =)

**Comment by laynetrain** [comment]
Toggleable bottom left of the UI?

**Comment by Raab70** [comment]
Duplicate of #295
